index.html

```diff
@@ -3,177 +3,11 @@
   <head>
 
         <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+        <meta name="viewport" content="width=device-width,initial-scale=1.0">
         <title>Space Empires 4x Production Sheet</title>
-        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/toastr.js/latest/css/toastr.min.css">
-        <style type="text/css">
-            .toast {
-                opacity: 1 !important;
-            }
-        </style>
   </head>
   <body>
-    <div id="app">
```

Comment:
Most of the content here was moved to either App.vue or the various XYZTab.vue files.